.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '0.10'
+  - '4.2.2'
```

Comment:
@sajinshrestha We will likely have users on everything from 0.10.x, or at the minimum 0.12.x that we need to support.  Thoughts on adding a range of environments to test against?